recipes/readstat/all/conanfile.py

```diff
@@ -0,0 +1,113 @@
+from conan.tools.microsoft import is_msvc, MSBuildDeps, MSBuildToolchain, MSBuild
+import os
+
+
+required_conan_version = ">=1.57.0"
+
+class ReadstatConan(ConanFile):
+    name = "readstat"
+    description = "Command-line tool (+ C library) for converting SAS, Stata, and SPSS files"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/WizardMac/ReadStat"
+    topics = ("spss", "stata", "sas", "sas7bdat", "readstats")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    @property
+    def _settings_build(self):
+        return getattr(self, "settings_build", self.settings)
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        # for plain C projects only
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("libiconv/1.17")
+
+    # if another tool than the compiler or autotools is required to build the project (pkgconf, bison, flex etc)
+    def build_requirements(self):
+        if self._settings_build.os == "Windows":
+            self.win_bash = True
+            if not self.conf.get("tools.microsoft.bash:path", check_type=str):
+                self.tool_requires("msys2/cci.latest")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        if is_msvc(self):
+            tc = MSBuildToolchain(self)
+            tc.generate()
+            deps = MSBuildDeps(self)
+            deps.generate()
+        else:
+            env = VirtualBuildEnv(self)
+            env.generate()
+            if not cross_building(self):
+                env = VirtualRunEnv(self)
+                env.generate(scope="build")
+            tc = AutotoolsToolchain(self)
+            tc.generate()
+            deps = AutotoolsDeps(self)
+            deps.generate()
+
+    def _msbuild_configuration(self):
+        return "Debug" if self.settings.build_type == "Debug" else "Release"
+    
+    def build(self):
+        if is_msvc(self):
+            msbuild = MSBuild(self)
+            msbuild.build_type = self._msbuild_configuration
+            msbuild.build(sln=os.path.join("VS17", "ReadStat.sln"))
+        else:
+            apply_conandata_patches(self)
+            autotools = Autotools(self)
+            autotools.configure()
+            autotools.make()
+            
+    def package(self):
```

Comment:
```suggestion
    def package(self):
        copy(self, "LICENSE", self.source_folder, os.path.join(self.package_folder, "licenses"))
```

Reply:
Thanks for your review!
btw: I have problems with msvc build not because I dont have that locally env.